content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/2. Uint.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 2. Uint
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Uint.OI
+  - ib-19.3.Datatype-Primitief-Uint.OI
+---
+
+### Wat is een Uint 
+Een Uint, ofwel unsigned integer, is een datatype die dezelfde ruimte in beslag neemt als de [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]]. De Uint kan alleen geen negatieve getallen opslaan, maar gebruikt deze ruimte om een twee keer zo groot positief getal op te kunnen slaan (4,294,967,295). 
+
+Voor de rest werkt de Uint hetzelfde als de [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]].
```

Comment:
Opmerking: heullll téchnisch gezien, hoort deze regel denk ik bij de 'hoe zit het in elkaar'. Denk niet dat het veel uitmaakt, echter.